phpunit/block-supports/layout-test.php

```diff
@@ -101,4 +102,302 @@ function test_outer_container_not_restored_for_aligned_image_block_with_themejso
+	 *
+	 *      @type string  $selector                      CSS selector.
+	 *      @type array   $layout                        Layout object. The one that is passed has already checked the existence of default block layout.
+	 *      @type boolean $has_block_gap_support         Whether the theme has support for the block gap.
+	 *      @type string  $gap_value                     The block gap value to apply.
+	 *      @type boolean $should_skip_gap_serialization Whether to skip applying the user-defined value set in the editor.
+	 *      @type string  $fallback_gap_value            The block gap value to apply.
+	 *      @type array   $block_spacing                 Custom spacing set on the block.
+	 * }
+	 * @param string $expected_output The expected output.
+	 */
+	function test_gutenberg_get_layout_style( $args, $expected_output ) {
+		$layout_styles = gutenberg_get_layout_style( $args['selector'], $args['layout'], $args['has_block_gap_support'], $args['gap_value'], $args['should_skip_gap_serialization'], $args['fallback_gap_value'], $args['block_spacing'] );
+		$this->assertSame( $expected_output, $layout_styles );
+	}
+
+	/**
+	 * Data provider for test_gutenberg_get_layout_style().
+	 *
+	 * @return array
+	 */
+	public function data_gutenberg_get_layout_style() {
+		return array(
+			'should_return_empty_value_with_no_args'       => array(
+				'args'            => array(
+					'selector'                      => null,
+					'layout'                        => null,
+					'has_block_gap_support'         => null,
+					'gap_value'                     => null,
+					'should_skip_gap_serialization' => null,
+					'fallback_gap_value'            => null,
+					'block_spacing'                 => null,
+				),
+				'expected_output' => '',
+			),
+			'should_return_empty_value_with_only_selector' => array(
+				'args'            => array(
+					'selector'                      => '.wp-layout',
+					'layout'                        => null,
+					'has_block_gap_support'         => null,
+					'gap_value'                     => null,
+					'should_skip_gap_serialization' => null,
+					'fallback_gap_value'            => null,
+					'block_spacing'                 => null,
+				),
+				'expected_output' => '',
+			),
+			'should_return_default_layout_with_block_gap_support' => array(
+				'args'            => array(
+					'selector'                      => '.wp-layout',
+					'layout'                        => null,
+					'has_block_gap_support'         => true,
+					'gap_value'                     => '1em',
+					'should_skip_gap_serialization' => null,
+					'fallback_gap_value'            => null,
+					'block_spacing'                 => null,
+				),
+				'expected_output' => '.wp-layout > *{margin-block-start:0;margin-block-end:0;}.wp-layout.wp-layout > * + *{margin-block-start:1em;margin-block-end:0;}',
+			),
+			'should_return_default_layout_with_block_gap_support_and_skip_serialization' => array(
```

Comment:
Just double-checking, but it looks like the test is correct but the name of the test should refer to it expecting an empty value?

```suggestion
			'should_return_empty_value_with_block_gap_support_and_skip_serialization' => array(
```